tests/slack_sdk/models/test_objects.py

```diff
@@ -148,6 +148,14 @@ def test_get_non_null_attributes_nested_2(self):
         )
         self.assertDictEqual(expected, nested.get_non_null_attributes())
 
+    def test_eq(self):
+        obj1 = SimpleJsonObject()
+        self.assertEqual(self.good_test_object, obj1)
+
+        obj2 = SimpleJsonObject()
+        obj2.test = "another"
+        self.assertNotEqual(self.good_test_object, obj2)
```

Comment:
This test code looks good. Can you add a few similar test patterns in `test_blocks.py` and `test_elements.py`, and `test_views.py`. `Block`, `BlockElement`, and `View` inherit `JsonObject`. We'd like to have the tests that verifies if the change works for the existing sub classes.